plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1116,19 +1110,25 @@ blockly.block.math_random_int_between=Random integer between min: %1 and max: %2
 blockly.block.math_random_number_between=Random number between min: %1 and max: %2
 blockly.block.mcitem_to_block=Convert %2 %1 to %3 or AIR if conversion fails
 blockly.block.move_entity=Set location of %4 to x: %1 y: %2 z: %3
-blockly.block.place_schematic=Place at x: %1 y: %2 z: %3 with rotation: %4 mirror: %5 structure:
+blockly.block.place_schematic=Place on server side at x: %1 y: %2 z: %3 with rotation: %4 mirror: %5 structure:
```

Comment:
We use `server-side` not `server side` in MCR

Reply:
I thought `server-side` should be used as adjective/adverb and `server side` - as noun with its own adjective.

Reply:
There is `Spawn %8 particles on server-side` and probably some others so if this is more grammatically correct, other procedure blocks should be corrected where appropriate to not mix the ways we use this